addon/templates/components/bootstrap-inputs/-date.hbs

```diff
@@ -30,7 +32,7 @@
       )
     }}
   {{else}}
-    {{control.label for=control.inputId}}
+    {{control.label}}
```

Comment:
We don't need the `for` attr?

Reply:
We do not. ControlWrapper sets those properties for us.